src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -348,40 +348,38 @@ namespace BaseboardElectric {
                                 "Baseboard Total Heating Energy",
                                 Constant::Units::J,
                                 thisBaseboard.Energy,
-                                OutputProcessor::SOVTimeStepType::System,
-                                OutputProcessor::SOVStoreType::Summed,
+                                OutputProcessor::TimeStepType::System,
```

Comment:
Collapsed indirection from `SOVTimeStepType` to `TimeStepType` and `SOVStoreType` to `StoreType`.